pkg/util/util.go

```diff
@@ -84,6 +84,17 @@ func FileExists(filename string) (bool, error) {
 	return true, nil
 }
 
+// Verify block device is not in use by calling openFile with O_EXCL flag.
+// Returns nil if device not in use, but a non-nil return value means
+// that either the device is in use or another error occurred during OpenFile.
+func CheckDeviceInUseBySystem(pathname string) error {
```

Comment:
Traditionally, what one does is try to interpret the error code here, in this function, and only return it as an error if it's one you don't understand or can't deal with. (bool, error) is the right return signature for a function like this.